subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/ivy/IvyFileModule.groovy

```diff
@@ -261,6 +277,44 @@ class IvyFileModule extends AbstractModule implements IvyModule {
         return this
     }
 
+    private void publishModuleMetadata() {
```

Comment:
:X: This duplication with `AbstractMavenModule` is a bit nasty. I want to add support for `excludes` and will have to do it in 2 places...

Reply:
fixed